glib/src/gobject/signal_group.rs

```diff
@@ -42,6 +42,50 @@ impl SignalGroup {
         self.connect_closure(signal_name, after, RustClosure::new_local(callback));
     }
 
+    #[inline]
+    pub fn connect_notify<F>(&self, name: Option<&str>, callback: F)
+    where
+        F: Fn(&Self, &crate::ParamSpec) + Send + Sync + 'static,
+    {
+        let signal_name = if let Some(name) = name {
+            format!("notify::{name}")
+        } else {
+            "notify".into()
+        };
+
+        let closure = crate::RustClosure::new(move |values| {
+            let obj = values[0].get().unwrap();
```

Comment:
Ah, it returns the target object instead of the SignalGroup. Not sure what to do here. I think we can either return a glib::Object of the target object or clone the SignalGroup and return that instead. I'm inclined for the former.

Reply:
I'll push that for now.

Reply:
Yes that's what I would expect. The other signals should work the same way